posthog/settings/data_stores.py

```diff
@@ -106,8 +106,8 @@
 
 _parse_kafka_hosts = lambda kafka_url: ",".join(urlparse(host).netloc for host in kafka_url.split(","))
 
-# URL Used by kafka clients/producers
-KAFKA_URL = os.getenv("KAFKA_URL", "kafka://kafka")
+# URL(s) used by Kafka clients/producers - KEEP IN SYNC WITH plugin-server/src/config/config.ts
+KAFKA_URL = os.getenv("KAFKA_URL", "kafka://kafka:9092")
```

Comment:
why are there these two places?

Reply:
Before this PR I just merged `KAFKA_HOSTS` in Python had a default, but it was a bit off because it didn't have the port, which is usually required. While the plugin server required `KAFKA_HOSTS` to be provided explicitly. Now this is 100% synced, for better development ergonomics (no need to set `KAFKA_HOSTS` anywhere in dev) and a more obvious relation between those codebases.